### PR TITLE
Campaign details block

### PR DESCRIPTION
**Pull request by mdickson-adbe**
Additional tweaks to campaign overview/deliverables info tabs for better alignment with mockup/MVP

Test URLs:
Before: https://main--assets-distribution-portal--adobe.hlx.page/sample-public-site
After: https://assets-98992--adobe-gmo--hlxsites.hlx.page/drafts/mdickson/campaign-details

### CODE REVIEW

File: blocks/gmo-campaign-details/gmo-campaign-details.js [hunk 1]
```diff
@@ -604,10 +698,25 @@ function buildTableRow(campaignJson, createHidden) {
         <div class='property table-column column1'>${campaignJson.name}</div>
         <div class='property table-column column2'>${campaignJson.type}</div>
         <div class='property table-column column3'>${campaignJson.channel}</div>
-        <div class='property table-column column4'>${campaignJson.reviewLinkName}</div>
-        <div class='property table-column column5'>${campaignJson.finalAssetName}</div>
+        <div class='property table-column column4'>
+            <a href="${campaignJson.reviewLinkHref}" class="campaign-link">${campaignJson.reviewLinkName}</a>
```
Comment (shiv-gup) [inline blocks/gmo-campaign-details/gmo-campaign-details.js:702]: @mdickson-adbe  lets hardcode the label "Review Link"

File: blocks/gmo-campaign-details/gmo-campaign-details.js [hunk 2]
```diff
@@ -604,10 +698,25 @@ function buildTableRow(campaignJson, createHidden) {
...
+        </div>
+        <div class='property table-column column5'>
+            <a href="${campaignJson.finalAssetHref}" class="campaign-link">${campaignJson.finalAssetName}</a>
```
Comment (shiv-gup) [inline blocks/gmo-campaign-details/gmo-campaign-details.js:705]: @mdickson-adbe lets hardcode the label to "Final Asset" only link will be provided from CF's

File: blocks/gmo-campaign-details/gmo-campaign-details.js [hunk 3]
```diff
@@ -418,6 +474,20 @@ export default async function decorate(block) {
                         <div class="use-case-tag">Use Case 2</div>
                     </div>
                 </div>
+                <div class="main-message-wrapper">
```
Comment (shiv-gup) [inline blocks/gmo-campaign-details/gmo-campaign-details.js:477]: @mdickson-adbe Can you comment out this section. we are planning to add this next phase 2.0

File: blocks/gmo-campaign-details/gmo-campaign-details.js [hunk 4]
```diff
@@ -418,6 +474,20 @@ export default async function decorate(block) {
...
+                    <span class="h3">Main Message</span>
+                    <span class="description">
+                        A major genAI release of the Photoshop beta app that delivers new and enhanced generative AI capabilities.
+                    </span>
+                </div>
+                <div class="channel-scope-wrapper">
```
Comment (shiv-gup) [inline blocks/gmo-campaign-details/gmo-campaign-details.js:483]: @mdickson-adbe Can you comment out this section "Key Channel Scope". we are planning to add this next phase 2.0

File: blocks/gmo-campaign-details/gmo-campaign-details.js [hunk 5]
```diff
@@ -453,12 +523,36 @@ export default async function decorate(block) {
                         Adobe Express Mobile App
                     </div>
                 </div>
+                <div class="card scope">
```
Comment (shiv-gup) [inline blocks/gmo-campaign-details/gmo-campaign-details.js:526]: @mdickson-adbe Can you comment out this section "Feature scope". we are planning to add this next phase 2.0

File: blocks/gmo-campaign-details/gmo-campaign-details.js (hunk 3 above)
Comment (mdickson-adbe) [reply blocks/gmo-campaign-details/gmo-campaign-details.js:477]: I'll hide it with CSS for now!

**Pull request merged**